client/src/main/java/tds/assessment/Item.java

```diff
@@ -222,21 +224,6 @@ public void setPrintable(boolean printable) {
     }
 
     /**
-     * Finds the language code of the item based on its item properties.
-     *
-     * @return The language code of the item if one exists in its properties - otherwise null
-     */
-    public String getLanguageCode() {
```

Comment:
This property isn't used anywhere.  TDS_ExamService and TDS_ExamResultsTransmitter both depend on the TDS_Assessment client library, and both built fine after removing this property (made sure to point their `pom.xml` files to use the 3.1.1-SNAPSHOT version of this library to be sure).